### DISCUSSION TITLE
Is it possible to have multiple instances of IMM open at the same time?

### DISCUSSION THREAD

**Discussion by axelNTI** [bcssov/IronyModManager#530] [Q&A] [answered]
I'm going through two playsets at once and would prefer if I could view both at the same. I am therefore wondering if it is possible to have multiple instances of IMM open at the same time? Launching the exe multiple times does not seem to work.

**Comment by bcssov** [comment, marked as answer]
Irony was never intended to have more than 1 instance, however since it was allowed for a long time you can set a flag:
https://github.com/bcssov/IronyModManager/blob/21d378461b914c1df14cfb5722bba54353efca61/src/IronyModManager/appSettings.json#L45
No support if anything breaks

**Reply by axelNTI** [reply]
It works great. Thanks for the help.
